dataactvalidator/config/sqlrules/c8_award_financial.sql

```diff
@@ -0,0 +1,13 @@
+SELECT af.row_number, af.fain, af.uri
+FROM award_financial AS af
+WHERE af.submission_id = {}
+	AND af.fain IS NOT NULL
+	AND af.uri IS NOT NULL
```

Comment:
This will only catch errors where a row has both fain and uri.  If it only has one of the two, that still needs to match the other file.